src/platforms/darwin.ts

```diff
@@ -82,7 +82,7 @@ export default class MacOSPlatform implements Platform {
 
   private isNSSInstalled() {
     try {
-      return run('brew list').toString().indexOf('nss') > -1;
+      return run('brew list').toString().split('\n').indexOf('nss') > -1;
```

Comment:
`brew list` supports a `-1` flag for printing each entry on one line, which you could use instead of the string manipulation here. That's a common flag for many shell commands, so it's unlikely to change.
```suggestion
      return run('brew list -1').toString().includes('\nnss\n');
```